thingif/src/main/java/com/kii/thingif/query/HistoryStatesQuery.java

```diff
@@ -2,15 +2,18 @@
 
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+import android.text.TextUtils;
 
+import com.google.gson.annotations.SerializedName;
 import com.kii.thingif.clause.query.QueryClause;
 
 public class HistoryStatesQuery {
 
-    private @NonNull String alias;
-    private @NonNull QueryClause clause;
+    private transient @NonNull String alias;
+    private transient @NonNull QueryClause clause;
```

Comment:
Why does this 2 fields becomes `transient`? 
The 2 field are required to serialize by Gson aren't they?